src/tile/tile.scss

```diff
@@ -313,3 +315,30 @@
     @include visually-hidden;
   }
 }
+
+@mixin iui-tile-array {
+  width: 100%;
+
+  // #region IE support
+  .iui-tile {
+    margin: $iui-baseline $iui-sm;
+  }
+
+  @supports (grid-gap: $iui-line-height $iui-l) {
+    .iui-tile {
+      margin: 0;
+    }
+  }
+  // #endregion IE support
+
+  @supports (grid-gap: $iui-line-height $iui-l) {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax($tile-min-width, 1fr));
+    grid-gap: $iui-line-height $iui-l;
+    align-items: stretch;
+
+    .iui-tile {
+      width: 100%;
```

Comment:
This shouldn't be needed ideally, as the grid is in charge of the tile width

Reply:
If I remove this the tiles keep their fixed width.
![Screen Shot 2022-02-17 at 10 17 41 AM](https://user-images.githubusercontent.com/849817/154512007-c186289f-5981-4f64-af58-cd350f021163.png)

Reply:
Yeah, I think you'd need to change the grid formula to account for tile's fixed width. Try this:
```css
grid-template-columns: repeat(auto-fit, minmax(min(100%, 288px), 1fr));
```

Reply:
This suggested code is giving me an error on build:
```
[scss] Error converting files to CSS: Error: Incompatible units: 'px' and '%'.
error Command failed with exit code 1.
```

Reply:
Seems to work in the browser so maybe this is a Sass problem again. Try hacking around the Sass min function like this:
```scss
grid-template-columns: repeat(auto-fit, minmax(unquote("min(100%, #{$tile-min-width})"), 1fr));
```

Reply:
Wow, replacing `min` with `Min` does let the code compile without error, but the tiles are still appearing like they did [above](https://github.com/iTwin/iTwinUI/pull/479#discussion_r809162689) with `.iui-tile { width: 100%; }` removed.

Reply:
Is now using `> *` and using a CSS variable.  Also renamed the mixin and class to `.iui-responsive-grid`.

Reply:
Maybe we should move it to a different file then? 🤔

Reply:
@gretanausedaite can you look into adding this to iTwinUI-Layouts?  Don't spend too much time on it as this decision isn't final but I'm certainly leaning more towards it after this morning's call.